userbenchmark/optim/run.py

```diff
@@ -377,14 +377,15 @@ def is_excluded(mn: str, d: str, on: str, func_str: str, defaults: Dict[str, Any
 def run_model(modelName, device, Optim, defaults, maybe_pt2_):
     try:
         params = get_model_params(modelName, device)
-        print('getting params: ', params[0].size(), params[0].dtype, len(params), params[0].device)
+        print(datetime.datetime.now(), 'getting params: ', params[0].size(), params[0].dtype, len(params), params[0].device)
```

Comment:
Maybe it is better to use some standard library like `logging` here?

Reply:
if i add another print statement i will switch to a logger, right now it doesn't seem to provide more value add.